matts_simple_sims/psf_homogenize/psf_homogenizer.py

```diff
@@ -0,0 +1,223 @@
+import numpy as np
```

Comment:
This is the one.

Reply:
what's the problem?

Reply:
There is no specific problem per say. The issue is simply to make it work better.

Reply:
If you look at this [notebook](https://github.com/esheldon/metadetect/blob/metadetect-and-cal/matts_simple_sims/psf_homogenize/test_psf_homogenizer_piff.ipynb), you can see at output 13 the residuals are pretty high still.

Reply:
This comment is marking which version of the PSF homogenization we are currently using. There are a lot of versions in this PR and I didn't want ppl to be confused.